contrib/pyln-grpc-proto/pyln/grpc/node_pb2_grpc.py

```diff
@@ -5,7 +5,7 @@
 
 from pyln.grpc import node_pb2 as node__pb2
 
-GRPC_GENERATED_VERSION = '1.75.1'
+GRPC_GENERATED_VERSION = '1.76.0'
```

Comment:
@Nazarevsky Thanks for the PR. Since this is your first PR, I will nitpick. 

The CI is currently failing due to version mismatch in your `grpcio`. Please downgrade your local grpcio (`uv pip install grpcio==1.75.1`).